mbuild/examples/alkane_monolayer/alkane_monolayer.py

```diff
@@ -28,16 +28,21 @@ def __init__(self, mask, tile_x=1, tile_y=1, chain_length=10):
         hydrogen = H()
 
         # Attach chains to specified binding sites. Other sites get a hydrogen.
-        mb.apply_mask(host=self.tiled_surface, guest=alkylsilane, mask=mask,
-                      backfill=hydrogen)
+        alkylsilanes, hydrogens = mb.apply_mask(host=self.tiled_surface,
+                                                guest=alkylsilane, mask=mask,
+                                                backfill=hydrogen)
+        self.add(alkylsilanes)
+        self.add(hydrogens)
 
 
 def main():
     from mbuild.mask import grid_mask_2d
     mask = grid_mask_2d(8, 8)  # Evenly spaced, 2D grid of points.
     monolayer = AlkaneMonolayer(chain_length=10, mask=mask)
+    monolayer.visualize()
+    monolayer.view_hierarchy()
```

Comment:
This will fail on Travis since we're treating `networkx` as an optional dependency. I suppose it is really easy to install so we could consider just making it required. Might come in handy down the road anyways. Thoughts @sallai ?